index.html

```diff
@@ -1,534 +1,1010 @@
+            juridică în școli”, demonstrându-și abilitatea de a explica și
+            dezbate conceptele juridice într-un mod accesibil pentru elevi.
+          </p>
         </div>
-        <div class="features__deatail-cont">
-          <h5 class="features__header">Cabinetul oferă servicii juridice</h5>
+        <div
+          class="operations__content operations__content--3 operations__content--active"
+        >
+          <div class="operations__icon operations__icon--3">
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              xmlns:xlink="http://www.w3.org/1999/xlink"
+              version="1.1"
+              x="0px"
+              y="0px"
+              viewBox="0 0 50 50"
+              style="enable-background: new 0 0 50 50"
+              xml:space="preserve"
+            >
+              <g id="Layer_1">
+                <path
+                  d="M45.281,25.915c4.949-5.004,4.949-13.146,0-18.15C42.881,5.337,39.688,4,36.292,4c-0.001,0-0.001,0-0.001,0   c-3.396,0-6.59,1.337-8.991,3.765L25,10.09l-2.3-2.325C20.299,5.337,17.106,4,13.709,4c-3.396,0-6.589,1.337-8.99,3.765   c-4.949,5.004-4.949,13.146,0,18.15L25,46.422L45.281,25.915z M6.141,9.171C8.163,7.126,10.852,6,13.709,6   c2.858,0,5.547,1.126,7.569,3.171L25,12.935l3.722-3.764C30.744,7.126,33.433,6,36.291,6c2.858,0,5.546,1.126,7.568,3.171   c4.183,4.229,4.183,11.109,0,15.338L25,43.578L6.141,24.509C1.958,20.28,1.958,13.399,6.141,9.171z"
+                ></path>
+              </g>
+              <g></g>
+            </svg>
+          </div>
+          <h5 class="operations__header">
+            Pasiune pentru justiție de calitate
+          </h5>
           <p>
-            Avocatul Sebastian Nagy Cabinetul oferă servicii juridice în
-            diverse domenii de drept, adaptate nevoilor specifice
-            dumneavoastră.
+            Pasiunea pentru un act de justiție de calitate l-a împins pe
+            Sebastian Nagy sa fie prezent în mod constant în presa locală și
+            națională, contribuind cu expertiza sa la educația juridică a
+            intregii societati.
           </p>
         </div>
       </div>
-      <img src="img/law-3--lazy.jpg" data-src="img/law-2.jpg" alt="Plant" class="features__img lazy-img" />
-      <div class="features__feature features__feature-m">
-        <div class="features__icon features__icon-m">
-          <svg>
-            <use xlink:href="img/icons.svg#icon-trending-up"></use>
-          </svg>
-        </div>
-        <div class="features__deatail-cont">
+    </section>
+
+    <section class="section" id="servicii">
+      <div class="section__title">
+        <h2 class="section__description">Soluții există</h2>
+        <h3 class="section__header">
+          Servicii juridice de încredere în Cluj-Napoca
+        </h3>
+      </div>
+
+      <div class="features">
+        <img
+          src="img/law-1.jpg"
+          data-src="img/law-1.jpg"
+          alt="Computer"
```

Comment:
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii".